dotcom-rendering/src/web/components/StickyBottomBanner.importable.tsx

```diff
@@ -207,7 +211,12 @@ export const StickyBottomBanner = ({
 	switches,
 	pageId,
 	keywordsId,
-}: Props) => {
+}: Props & {
+	switches: Switches;
```

Comment:
Having had a second look at the uses of this switches array, it seems like they're used to look for the keys `puzzlesBanner` and `remoteBanner`, which are not AB switches, but on line 339 we pass in `abTestSwitches`. 

I think if https://github.com/guardian/dotcom-rendering/pull/4190 then the switches required would then never be available